webpack.config.js

```diff
@@ -11,6 +12,10 @@ const baseConfig = {
   module: {
     loaders: [
       {
+        test: /\.css$/,
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'),
```

Comment:
What's the assets releasing process? I'm vaguely remembering we spoke about compiling and pushing all assets (css, images) as part of a lambda deploy to a separate S3 bucket.